script.js

```diff
@@ -0,0 +1,60 @@
+const CLAIM_BUTTON = document.getElementById("submitBtn");
+
+const INPUT_PASSWORD = document.getElementById("password");
+const INPUT_EMAIL = document.getElementById("email");
+const INPUT_LNAME = document.getElementById("lName");
+const INPUT_FNAME = document.getElementById("fName");
```

Comment:
I would refer to these elements as the "Error messages"

```suggestion
const PASSWORD_ERROR_MESSAGE = document.getElementById("passwordErrorMessage");
const EMAIL_ERROR_MESSAGE = document.getElementById("emailErrorMessage");
const LNAME_ERROR_MESSAGE= document.getElementById("lNameErrorMessage");
const FNAME_ERROR_MESSAGE= document.getElementById("fNameErrorMessage");
```

**Note:** Your HTML will have to get updated as well